src/PhpBrew/Patches/PHP56WithOpenSSL11Patch.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace PhpBrew\Patches;
+
+use PhpBrew\Buildable;
+use PhpBrew\PatchKit\Patch;
+use PhpBrew\PatchKit\DiffPatchRule;
+use CLIFramework\Logger;
+
+class PHP56WithOpenSSL11Patch extends Patch
+{
+    public function desc()
+    {
+        return 'php5.6 with openssl 1.1.x patch.';
+    }
+
+    public function match(Buildable $build, Logger $logger)
+    {
+        return true;
```

Comment:
This needs to be removed.

Reply:
Looks like this conversation was resolved mistakenly. We cannot accept this PR in its current state.